packages/amplify-graphql-auth-transformer/src/vtl-generator/rds/resolvers/common.ts

```diff
@@ -0,0 +1,9 @@
+import { compoundExpression, methodCall, obj, printBlock, ref, toJson } from 'graphql-mapping-template';
+
+/**
+ * Generates default RDS expression
+ */
+export const generateDefaultRDSExpression = (): string => {
+  const exp = methodCall(ref('util.unauthorized'));
+  return printBlock('Default RDS Auth Resolver')(compoundExpression([exp, toJson(obj({}))]));
```

Comment:
There's an awful lot of punctuation in this, making it pretty hard to understand what's going on. If this is intended to be a final implementation, could you split out some interim variables to make the intent of these things clearer?

Reply:
Right now, it is meant to produce dummy VTL for RDS models. However this is kind of pretty common in VTL generation code throughout our code base.